### DISCUSSION TITLE
Can I play Lego builder's journey on it?

### DISCUSSION THREAD

**Discussion by Amidst1** [skyline-emu/skyline#317] [Q&A] [answered]
Can I play Lego builder's journey on it? probably a stupid question but, will it work? Cuz skyline doesn't have graphics yet so will it work?

**Comment by PixelyIon** [comment, marked as answer]
It may work if it doesn't do anything too fancy but if it does then we're probably not going to take a look at it for a long time, it's not particularly a game anyone in the team is interested in as far as I'm aware.

**Reply by Amidst1** [reply]
I'm very much interested :P

**Comment by Amidst1** [comment]
Update: i tried to play a simple orthographic game to see if the further could work. I tried hocus 2, but it crashes, so that either means that it's because of the graphic lack or my chipset is using Mali GPU.
Lego builder's journey won't work on such an early stage.
